release/rancher/rancher.go

```diff
@@ -108,6 +124,31 @@ type registryAuthToken struct {
 	Token string `json:"token"`
 }
 
+type regsyncConfig struct {
+	Version  int             `yaml:"version"`
+	Creds    []regsyncCreds  `yaml:"creds"`
+	Defaults regsyncDefaults `yaml:"defaults"`
+	Sync     []regsyncSync   `yaml:"sync"`
+}
```

Comment:
Not sure if gofmt doesn't add new lines or not but let's let this code breathe a little and space out the structs and functions.